packages/app/src/debug/DebugRunDetailedView.vue

```diff
@@ -0,0 +1,253 @@
+                <LightText>
+                  {{ specsCompleted(run) }}
+                </LightText>
+              </div>
+
+              <LightText>
+                {{ formatDuration(run.totalDuration ?? 0) }} ({{ formatCreatedAt(run.createdAt) }})
+              </LightText>
+            </div>
+          </li>
+        </ul>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { gql, useMutation } from '@urql/vue'
+import Button from '@packages/frontend-shared/src/components/Button.vue'
+import { groupBy } from 'lodash'
+import { computed, FunctionalComponent, h, ref } from 'vue'
+import type { DebugRunDetailedViewFragment, DebugRunDetailedRunInfoFragment } from '../generated/graphql'
+import { DebugRunDetailedView_MoveToRunDocument } from '../generated/graphql'
+import { formatDuration, formatCreatedAt } from './utils/formatTime'
+import DebugResults from './DebugResults.vue'
+import DebugRunNumber from './DebugRunNumber.vue'
+import DebugCommitIcon from './DebugCommitIcon.vue'
+import DebugCurrentRunIcon from './DebugCurrentRunIcon.vue'
+import { IconChevronDownLarge, IconChevronRightLarge } from '@cypress-design/vue-icon'
+
+gql`
+fragment DebugRunDetailedRunInfo on CloudRun {
+  __typename
+  runNumber
+  totalTests
+  totalFailed
+  totalPassed
+  totalPending
+  totalSkipped
+  totalDuration
+  totalFlakyTests
+  totalInstanceCount
+  completedInstanceCount
+  id
+  status
+  specs {
+    id
+    path
+  }
+  createdAt
+  commitInfo {
+    sha
+    summary
+  }
+}
+`
+
+// all: runsByCommitShas(commitShas: ["fea0b14c3902050ee7962a60e01b0d53d336d589", "f5a499232263f6e6a6aac77ce05ea09cf4b4aad8"]) {
+gql`
+fragment DebugRunDetailedView on Query {
+  currentProject {
+    id
+    projectId
+    cloudProject {
+      __typename
+      ... on CloudProject {
+        id
+        all: runsByCommitShas(commitShas: $commitShas) {
+          id
+          ...DebugRunDetailedRunInfo
+        }
+        current: runByNumber(runNumber: $runNumber) {
+          id
+          ...DebugRunDetailedRunInfo
+        }
+      }
+    }
+  }
+}
+`
+
+gql`
+mutation DebugRunDetailedView_moveToRun($runNumber: Int!) {
+  moveToRelevantRun(runNumber: $runNumber)
+}
+`
+
+const props = defineProps<{
+  gql: DebugRunDetailedViewFragment
+}>()
+
+const Dot: FunctionalComponent = () => {
+  return h('span', { class: 'px-8px' }, '•')
+}
+
+const LightText: FunctionalComponent = (_props, { slots }) => {
+  return h('span', { class: 'text-sm text-gray-700' }, slots?.default?.())
+}
+
+const showRuns = ref(true)
+
+const cloudProject = computed(() => {
+  return props.gql?.currentProject?.cloudProject?.__typename === 'CloudProject' ? props.gql.currentProject.cloudProject : null
+})
+
+const latest = computed(() => cloudProject.value?.all?.[0])
+
+const current = computed(() => cloudProject.value?.current)
+
+const latestIsCurrentlySelected = computed(() => {
+  return latest.value?.commitInfo?.sha === current.value?.commitInfo?.sha
```

Comment:
This should be comparing run numbers, not shas here. 
```suggestion
  return latest.value?.runNumber === current.value?.runNumber
```

Reply:
Thanks!